vendor/bat-native-ads/src/bat/ads/internal/account/ad_rewards/ad_rewards.cc

```diff
@@ -283,7 +283,8 @@ void AdRewards::OnGetPayments(const UrlResponse& url_response) {
   }
 
   if (!payments_->SetFromJson(url_response.body)) {
-    BLOG(0, "Failed to parse payment balance: " << url_response.body);
+    BLOG(0, "Failed to parse payment balance");
+    BLOG(1, "Payment balance response body: " << url_response.body);
```

Comment:
Elsewhere in ads and ledger we have used `6` for responses, would be good to use the same value so that these are only shown when the verbosity is 6 or higher.